content/hi/docs/reference/glossary/pod-lifecycle.md

```diff
@@ -0,0 +1,19 @@
+---
+title: पॉड जीवनचक्र (Pod Lifecycle)
+id: pod-lifecycle
+date: 2019-02-17
+full-link: /docs/concepts/workloads/pods/pod-lifecycle/
+related:
+ - pod
+ - container
+tags:
+ - fundamental
+short_description: >
+  अवस्थाओं का क्रम जिसके माध्यम से एक पॉड अपने जीवनकाल में गुजरता है।
+ 
+---
+ अवस्थाओं का क्रम जिसके माध्यम से एक पॉड अपने जीवनकाल में गुजरता है।
+
+<!--अधिक-->
+
+[पॉड जीवनचक्र](/docs/concepts/workloads/pods/pod-lifecycle/) को पॉड की अवस्थाओं या चरणों द्वारा परिभाषित किया जाता है। पाँच संभावित पॉड चरण हैं: Pending, Running, Succeeded, Failed और Unknown। पॉड स्थिति का एक उच्च-स्तरीय विवरण [पॉडस्टैटस](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podstatus-v1-core) `phase` फ़ील्ड में सारांशित किया गया है। .
```

Comment:
@sftim : Sorry to bother you. Would you be able to comment if the pod states need to be translated or kept as-is?
```suggestion
[पॉड जीवनचक्र](/docs/concepts/workloads/pods/pod-lifecycle/) को पॉड की अवस्थाओं या चरणों द्वारा परिभाषित किया जाता है। पाँच संभावित पॉड चरण हैं: Pending, Running, Succeeded, Failed और Unknown। पॉड स्थिति का एक उच्च-स्तरीय विवरण [पॉडस्टैटस](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podstatus-v1-core) `phase` फ़ील्ड में सारांशित किया गया है। .
```

Reply:
We should leave `Pending`, `Running`, `Succeeded`, `Failed` in English as they are part of the API. I personally would follow each of those with an explanation in Hindi of what these terms represent.

Also I would write them inside code backticks, which is fine to do even if the original page doesn't do that. Localization is more than just translation.

Reply:
@surbhiahuja : Whenever you get the time, could you please open an issue with links to these comments so that we can work on them later? I'll go ahead with merging this change now.